docs/design/README.md

```diff
@@ -3053,18 +3180,18 @@ Carbon generics have a number of other features, including:
     same data representation as an existing type, so you may cast between the
     two types, but can implement different interfaces or implement interfaces
     differently.
--   Additional requirements can be placed on the associated types of an
+-   Additional requirements can be placed on the associated constants of an
     interface using
     [`where` constraints](generics/details.md#where-constraints).
```

Comment:
I think this is currently only possible for associated facets, because `where` produces a facet type. Maybe:
```suggestion
-   Additional requirements can be placed on the associated facets of an
    interface using
    [`where` constraints](generics/details.md#where-constraints).
```